src/elbo.jl

```diff
@@ -0,0 +1,235 @@
+"Likelihoods which take a scalar (or vector of scalars) as input and return a single scalar."
+ScalarLikelihood = Union{BernoulliLikelihood,PoissonLikelihood,GaussianLikelihood}
+
+
+abstract type ExpectationMethod end
+struct Default <: ExpectationMethod end
+struct Analytic <: ExpectationMethod end
+
+struct GaussHermite <: ExpectationMethod
```

Comment:
Maybe `Quadrature` would be more adapted, especially if we want to consider different approaches than `GaussHermite` later

Reply:
That makes sense - also, do you think I should call them `DefaultExpectation`, `AnalyticExpectation` etc... instead or leave them as they are?

Reply:
Personally, I think this is fine for now. Lets just get this merged. We can break / generalise it later if needed.